docroot/modules/features/resource/config/optional/views.view.mirador_view.yml

```diff
@@ -0,0 +1,267 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.node.mirador
+    - field.storage.node.field_mi
+    - node.type.resource
+  module:
+    - node
+    - user
+id: mirador_view
+label: 'Mirador View'
+module: views
+description: ''
+tag: ''
+base_table: node_field_data
+base_field: nid
+core: 8.x
+display:
+  default:
+    display_plugin: default
+    id: default
+    display_title: Master
+    position: 0
+    display_options:
+      access:
+        type: perm
+        options:
+          perm: 'access content'
+      cache:
+        type: tag
+        options: {  }
+      query:
+        type: views_query
+        options:
+          disable_sql_rewrite: false
+          distinct: false
+          replica: false
+          query_comment: ''
+          query_tags: {  }
+      exposed_form:
+        type: basic
+        options:
+          submit_button: Apply
+          reset_button: false
+          reset_button_label: Reset
+          exposed_sorts_label: 'Sort by'
+          expose_sort_order: true
+          sort_asc_label: Asc
+          sort_desc_label: Desc
+      pager:
+        type: some
+        options:
+          items_per_page: 10
+          offset: 0
+      style:
+        type: html_list
+      row:
+        type: 'entity:node'
+        options:
+          relationship: none
+          view_mode: mirador
+      fields:
+        field_mi:
+          id: field_mi
```

Comment:
In fact, do we really need to render fields in the view. Shouldn't this be content?